pkg/imagebuilder/report_test.go

```diff
@@ -19,6 +19,7 @@ Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/p
 ---IMAGE BUILD REPORT---
 {
     "status": "Succeeded",
+		"pushed": true,
```

Comment:
It looks like there is to many white characters in this line.

Reply:
In VS Code it's looking right, I don't knwo what happened here...

Reply:
ANyway, fixed